src/content/ja/2020/markup.md

```diff
@@ -0,0 +1,1342 @@
+---
+#See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
+title: マークアップ
+description: 2020年版Web Almanacのマークアップの章では、一般的な見解、要素や属性の使い方、トリビアやトレンドなどを紹介しています。
+authors: [j9t, catalinred, iandevlin]
+j9t_bio: Jens Oliver Meiert はウェブ開発者であり、著者(<a hreflang="en" href="https://leanpub.com/css-optimization-basics"><cite>CSS Optimization Basics</cite></a>, <a hreflang="en" href="https://leanpub.com/web-development-glossary"><cite>The Web Development Glossary</cite></a>)で、<a hreflang="en" href="https://www.jimdo.com/">Jimdo</a>でエンジニアリングマネージャーを務めています。彼は、HTMLとCSSの最適化を専門とするウェブ開発のエキスパートです。Jensは技術標準に貢献し、特に自身のウェブサイト<a hreflang="en" href="https://meiert.com/en/">meiert.com</a>では、仕事や研究について定期的に執筆しています。
+catalinred_bio: Catalin Rosuは、<a hreflang="en" href="https://www.caphyon.com/">Caphyon</a>のフロントエンド開発者で、現在は<a hreflang="en" href="https://www.wattspeed.com/">Wattspeed</a>に携わっています。ウェブ標準への情熱と、優れたUXとUIへの鋭い目を持ち、<a href="https://twitter.com/catalinred">ツイート</a>したり、<a hreflang="en" href="https://catalin.red/">自身のウェブサイト</a>で執筆したりしています。
+iandevlin_bio: Ian Devlinは、優れたセマンティックHTMLやアクセシビリティを提唱するウェブ開発者です。<a hreflang="en" href="https://www.peachpit.com/store/html5-multimedia-develop-and-design-9780321793935">HTML5 マルチメディア</a>についての本を書いたこともありますし、<a hreflang="en" href="https://iandevlin.com/">自身のウェブサイト</a>でウェブやその他のことについて散発的に書いています。現在は、ドイツの<a hreflang="de" href="https://www.real-digital.de/">real.digital</a>でシニアフロントエンドエンジニアとして働いています。
+reviewers: [zcorpan, matuzo, bkardell]
+analysts: [Tiggerito]
+editors: [rviscomi]
+translators: [ksakae]
+discuss: 2039
+results: https://docs.google.com/spreadsheets/d/1Ta7amoUeaL4pILhWzH-SCzMX9PsZeb1x_mwrX2C4eY8/
+featured_quote: 私たちは、生きたHTMLをほぼ完全に採用しようとしており、流行のページはすぐに削除し、フレームワークの採用や廃止も迅速に行っています。しかし、HTMLが与えてくれる選択肢を使い切った形跡はありません。
+featured_stat_1: 85.73%
+featured_stat_label_1: "living" HTML doctypeを使用しているページの割合
```

Comment:
What's your thoughts on whether we should translate "living" or not? Here and else where in this chapter when we talk about living HTML spec?

Reply:
Oh...
Sorry, I translate.